static/app/views/performance/newTraceDetails/traceDrawer/details/transaction.tsx

```diff
@@ -327,6 +327,9 @@ export function TransactionNodeDetails({
             <TraceDrawerComponents.Duration
               duration={durationInSeconds}
               baseline={avgDurationInSeconds}
+              baseDescription={
+                'Average duration for this transaction across its project, over the last 24 hours'
```

Comment:
```suggestion
                'Average duration for this transaction over the last 24 hours'
```
It is implied that it belongs to the project so lets not complicate :)